doc/whatsnew/2.14.rst

```diff
@@ -273,41 +294,50 @@ Other Changes
 
 * The ``method-cache-max-size-none`` checker will now also check ``functools.cache``.
 
-  Closes #5670
+* The ``config`` attribute of ``PyLinter`` is now of the ``argparse.Namespace`` type instead of
+  ``optparse.Values``.
 
-* ``BaseChecker`` classes now require the ``linter`` argument to be passed.
+  Ref #5392
 
-* The ``set_config_directly`` decorator has been removed.
+* ``UnsupportedAction`` has been deprecated.
 
-* The ``ignore-mixin-members`` option has been deprecated. You should now use the new
-  ``ignored-checks-for-mixins`` option.
+  Ref #5392
 
-  Closes #5205
+* ``OptionsManagerMixIn`` has been deprecated.
 
-* Fix falsely issuing ``useless-suppression`` on the ``wrong-import-position`` checker.
+  Ref #5392
 
-  Closes #5219
+* ``OptionParser`` has been deprecated.
 
-* Fixed false positive ``no-member`` for Enums with self-defined members.
+  Ref #5392
 
-  Closes #5138
+* ``Option`` has been deprecated.
 
-* Fix false negative for ``no-member`` when attempting to assign an instance
-  attribute to itself without any prior assignment.
+  Ref #5392
 
-  Closes #1555
+* ``OptionsProviderMixIn`` has been deprecated.
 
-* Changed message type from ``redefined-outer-name`` to ``redefined-loop-name``
-  (optional extension) for redefinitions of outer loop variables by inner loops.
+  Ref #5392
 
-  Closes #5608
+* ``ConfigurationMixIn`` has been deprecated.
 
-* By default the similarity checker will now ignore imports and ignore function signatures when computing
-  duplication. If you want to keep the previous behaviour set ``ignore-imports`` and ``ignore-signatures`` to ``False``.
+* The ``option_groups`` attribute of ``PyLinter`` has been deprecated.
 
-* Pylint now expands the user path (i.e. ``~`` to ``home/yusef/``) and expands environment variables (i.e. ``home/$USER/$project``
-  to ``home/yusef/pylint`` for ``USER=yusef`` and ``project=pylint``) for pyreverse's ``output-directory``,
-  ``import-graph``, ``ext-import-graph``,  ``int-import-graph`` options, and the spell checker's ``spelling-private-dict-file``
-  option.
+  Ref #5392
 
-  Relates to #6493
+* ``get_global_config`` has been deprecated. You can now access all global options from
+  ``checker.linter.config``.
+
+  Ref #5392
+
+* ``OptionsManagerMixIn`` has been replaced with ``ArgumentsManager``. ``ArgumentsManager`` is considered
+  private API and most methods that were public on ``OptionsManagerMixIn`` have now been deprecated and will
+  be removed in a future release.
+
+  Ref #5392
+
+* ``OptionsProviderMixIn`` has been replaced with ``ArgumentsProvider``. ``ArgumentsProvider`` is considered
+  private API and most methods that were public on ``OptionsProviderMixIn`` have now been deprecated and will
+  be removed in a future release.
+
+  Ref #5392
```

Comment:
This `diff` is horrible, but I'm just going to assume you copied these correctly 😄